scripts/generate_duplicate_sheet.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+
+import csv
+import itertools
+from datetime import datetime, timedelta
+
+from bugbug import bugzilla
+from bugbug.models.duplicate import DuplicateModel
+
+m = DuplicateModel()
+m.train()
+
+
+REPORTERS_TO_IGNORE = {"intermittent-bug-filer@mozilla.bugs", "wptsync@mozilla.bugs"}
+delta = timedelta(days=21)
```

Comment:
Nit: just use `datetime.now() - timedelta(days=21)` and remove the delta variable